src/components/TrafimageMaps/TrafimageMaps.js

```diff
@@ -203,10 +203,10 @@ const defaultProps = {
   initialState: {},
   menus: null,
   subMenus: null,
-  cartaroUrl: null,
-  geoServerUrl: null,
-  vectorTilesKey: null,
-  vectorTilesUrl: null,
+  cartaroUrl: process.env.REACT_APP_CARTARO_URL,
+  geoServerUrl: process.env.REACT_APP_GEOSERVER_URL,
+  vectorTilesKey: process.env.REACT_APP_VECTOR_TILES_KEY,
+  vectorTilesUrl: process.env.REACT_APP_VECTOR_TILES_URL,
```

Comment:
Can't we put prod urls by default? Don't forget  the app and the web-compoent will be separate. So these variables will not exist.

Reply:
or may be they will also exits in the web component build.

Reply:
THE web component would need to support the .env file. In that case it's the same as putting prod urls by default in here.

Reply:
yes